packages/migration/src/utils/elasticsearch-helper.ts

```diff
@@ -27,7 +27,6 @@ export const updateComposition = async (
   try {
     response = await client.update({
       index: ELASTICSEARCH_INDEX_NAME,
-      type: 'compositions',
```

Comment:
I still see a few references in the codebase for this. Should we remove them all now that we're on it?

Reply:
Yeah we can